<ui version="4.0" >
 <class>TTMpeg2MainWnd</class>
 <widget class="QMainWindow" name="TTMpeg2MainWnd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item rowspan="2" row="2" column="1" >
     <widget class="QGroupBox" name="gbVideoView" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="title" >
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="TTMPEG2Window" name="currentFrame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="TTStepBy" name="stepControl" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>248</width>
        <height>122</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>248</width>
        <height>122</height>
       </size>
      </property>
      <property name="title" >
       <string>Step by</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2" >
     <widget class="QSlider" name="scroller" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="TTVideoFileInfo" name="videoFileInfo" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>640</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title" >
       <string>Videofile</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2" >
     <widget class="TTFrameInfo" name="frameInfo" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>640</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title" >
       <string>Current Frame Info</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QGroupBox" name="gbInformation" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>248</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>248</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title" >
       <string>Information</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTabWidget" name="tabInformation" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabBuffer" >
          <attribute name="title" >
           <string>Buffer</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabHeader" >
          <attribute name="title" >
           <string>Header</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionFileOpen" />
    <addaction name="actionAnalyze" />
    <addaction name="separator" />
    <addaction name="actionSettings" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuExtra" >
    <property name="title" >
     <string>Extras</string>
    </property>
    <addaction name="actionWriteHeaderList" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuExtra" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionExit" />
   <addaction name="separator" />
   <addaction name="actionFileOpen" />
   <addaction name="actionAnalyze" />
   <addaction name="actionSettings" />
  </widget>
  <action name="actionFileOpen" >
   <property name="icon" >
    <iconset resource="ttmpeg2_mainwnd.qrc" >:/pixmaps/pixmaps/fileopen_24.xpm</iconset>
   </property>
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionAnalyze" >
   <property name="icon" >
    <iconset resource="ttmpeg2_mainwnd.qrc" >:/pixmaps/pixmaps/find.png</iconset>
   </property>
   <property name="text" >
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="ttmpeg2_mainwnd.qrc" >:/pixmaps/pixmaps/settings_16.xpm</iconset>
   </property>
   <property name="text" >
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="ttmpeg2_mainwnd.qrc" >:/pixmaps/pixmaps/exit_16.xpm</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWriteHeaderList" >
   <property name="text" >
    <string>Write header list...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTStepBy</class>
   <extends>QGroupBox</extends>
   <header>../ttmpeg2_src/gui/ttstepby.h</header>
  </customwidget>
  <customwidget>
   <class>TTFrameInfo</class>
   <extends>QGroupBox</extends>
   <header>../ttmpeg2_src/gui/ttframeinfo.h</header>
  </customwidget>
  <customwidget>
   <class>TTVideoFileInfo</class>
   <extends>QGroupBox</extends>
   <header>../gui/ttvideofileinfo.h</header>
  </customwidget>
  <customwidget>
   <class>TTMPEG2Window</class>
   <extends>QFrame</extends>
   <header>../mpeg2window/ttmpeg2window.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ttmpeg2_mainwnd.qrc" />
 </resources>
 <connections/>
</ui>
